Robinhood/Robinhood.py

```diff
@@ -326,16 +386,58 @@ def get_quote(self, stock=''):
         """Wrapper for quote_data """
 
         data = self.quote_data(stock)
-        return data["symbol"]
+        return data
 
-    def get_historical_quotes(self, stock, interval, span, bounds=Bounds.REGULAR):
-        """Fetch historical data for stock
+    def get_quotes_fields(self, stocks=None, fields=''):
+        """Returns multiple stock info and fields from quote_data
 
-            Note: valid interval/span configs
-                interval = 5minute | 10minute + span = day, week
-                interval = day + span = year
-                interval = week
-                TODO: NEEDS TESTS
+        Args:
+            stock (str|list): stock ticker(s) can be one or a list
+            fields (str|list): fields of the quote data that should return
+
+        Returns:
+            (:obj:`dict`): Dict with dict of the requested field values for stock
+        """
+        data = self.quotes_data(stocks)
+
+        # XXX Arguably fields should also be a list of Enum to catch typos
+        if isinstance(fields,str):
+            fields = [fields]
+
+        res = {}
```

Comment:
style overloaded.  `res` has been a requests/session object, and now it's a dict of values?

Reply:
It's largely been the 'response' return of the call.